Bowling/NUnitTestBowling/GameTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using BowlingApp;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+
+namespace NUnitTestBowling
+{   
+    public class DummyUserInteraction : IUserInteraction
+    {
+        public string BowlingMarks { get; set; }
+
+        public string GetBowlingMarks()
+        {
+            return BowlingMarks; 
+        }
+    }
+
+    class GameTests
+    {
+        private Game _subject;
+        private IUserInteraction _dummyUserInteraction;
+
+        public GameTests()
+        {
+            _dummyUserInteraction = new DummyUserInteraction();
+            _subject = new Game(_dummyUserInteraction);
+        }
+
+        [Test]
+        public void PlayGame_GivenPerfectGame_ReturnsPerfectScore()
+        {
+            _dummyUserInteraction.BowlingMarks = "XXXXXXXXXXXX";
+            _dummyUserInteraction.GetBowlingMarks();
```

Comment:
YES!  Looks like you figured this out! 😎

Reply:
This particular line actually isn't necessary since it's just fetching the value you just set, and not storing that value anywhere